master-firmware/src/strategy/goals.h

```diff
@@ -11,24 +11,31 @@ struct InitGoal : goap::Goal<RobotState> {
     }
 };
 
-struct RedPucksGoal : goap::Goal<RobotState> {
+struct ClassifyStartPucksGoal : goap::Goal<RobotState> {
     virtual int distance_to(const RobotState& state) const
     {
-        return goap::Distance().shouldBeEqual(state.pucks_in_deposit_zone[PuckColor_RED], 2);
+        return goap::Distance().shouldBeEqual(state.classified_pucks[PuckColor_REDGREEN], 3);
```

Comment:
Caution, those types of goals will fail completely if the robot can only find 2 pucks for example.

Reply:
I agree, I'm going to change the goals again over this week